tools/scripts/add-package-json-version.mjs

```diff
@@ -0,0 +1,20 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+    writePackageJson, parsePackageJson, getOutputPath
+} from './shared-util.mjs';
+import { workspaceRoot } from "@nrwl/devkit";
+
+// Executing this script: node path/to/add-package-json-version.mjs {projectName}
+const [, , projectName] = process.argv;
+
+process.chdir(workspaceRoot);
+const rootPackageJson = parsePackageJson();
+
+process.chdir(getOutputPath(projectName));
+
+const packageJson = parsePackageJson();
+packageJson.version = rootPackageJson.version;
+writePackageJson(packageJson);
```

Comment:
My understanding: write to the `package.json` in the output directory, so not into the source one. Would we need to write into the package.lock there as well? Or does it not exist in the output dir?

Reply:
You're right with your understanding. `package-lock.json` files don't exist in the output dirs.